RELEASE_NOTES.md

```diff
@@ -1,3 +1,66 @@
+## Cookstyle 6.1
+
+### 5 New Cops
+
+#### ChefModernize/NodeRolesInclude
+
+The `ChefModernize/NodeRolesInclude` cop detects cookbooks that use `node['roles'].include?('foo')` to see if a particular role has been applied to a node. The simpler `node.role?('foo')` helper should be used instead.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefDeprecations/PowershellCookbookHelpers
+
+The `ChefDeprecations/PowershellCookbookHelpers` cop detects cookbooks that use the deprecated `Powershell::VersionHelper.powershell_version?` helper from the `powershell` cookbook. Chef Infra Client provides information on the installed PowerShell release at `node['powershell']['version']` and Chef Infra Client 15.8 and later includes a new `powershell_version` which should be used when possible.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefCorrectness/ConditionalRubyShellout
+
+The `ChefCorrectness/ConditionalRubyShellout` detects resources that use Ruby code to shellout in not_if and only_if conditionals when they can just shellout without the additional complexity of Ruby.
```

Comment:
```suggestion
The `ChefCorrectness/ConditionalRubyShellout` cop detects resources that use unnecessary Ruby code in `not_if` and `only_if` conditionals to shellout when a string can be used to avoid additional complexity.
```

This sentence read weirdly to me and this was my best attempt to make it sound less awkward.